my_list.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './my_enumerable'
+
+class MyList
+  attr_accessor :list
+
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Kindly make sure to create your own `each` method that yields all members of @list in succession. (You should not use the ruby `each` method but create your own)